CPP_STYLE_GUIDE.md

```diff
@@ -33,6 +34,21 @@ these rules:
 
 `char* buffer;` instead of `char *buffer;`
 
+## C++ style comments
+
+Always use C++ style comments (`//`) for single line and multi-line comments.
```

Comment:
I would remove the `Always`, and add a comment that there might still be "old" comment in the code:
```markdown
Use C++ style comments (`//`) for both single line and multi-line comments.

Examples:
...

The codebase may contain old C style comments (`/* */`) from before this was the prefered style.
```